src/lib/datatypes/include/sol-buffer.h

```diff
@@ -420,6 +420,17 @@ int sol_buffer_append_slice(struct sol_buffer *buf, const struct sol_str_slice s
 int sol_buffer_append_bytes(struct sol_buffer *buf, const uint8_t *bytes, size_t size);
 
 /**
+ * @brief Appends the buffer @c from in the end of @c buf,
+ * reallocating if necessary.
```

Comment:
I'd do "s/Appends the/Appends the contents of the/", "s/in/at" and s/reallocating if/reallocating @c if/".